modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/exportimport/data/handler/test/BatchEnginePortletDataHandlerTest.java

```diff
@@ -223,6 +234,66 @@ public void testExportImportCompanyGroupObjectEntriesWithError()
 					objectDefinition.getName())));
 	}
 
+	@Ignore("LPD-40798")
+	@Test
+	@TestInfo("LPD-57756")
+	public void testExportImportPriorityWithSiteScopeObjectEntries()
+		throws Exception {
+
+		ObjectDefinition objectDefinition = _addObjectDefinition(
+			ObjectDefinitionConstants.SCOPE_SITE);
+
+		Group group1 = GroupTestUtil.addGroup();
+
+		Layout layout = LayoutTestUtil.addTypePortletLayout(group1);
+
+		LayoutTestUtil.addPortletToLayout(
+			layout, JournalContentPortletKeys.JOURNAL_CONTENT);
+
+		ObjectEntry[] objectEntries = _addObjectEntries(
+			3, group1.getGroupId(), objectDefinition);
+
+		File larFile = _exportLayouts(
+			false, group1.getGroupId(), false,
+			new long[] {layout.getLayoutId()}, objectDefinition);
+
+		Group group2 = GroupTestUtil.addGroup();
+
+		BundleContext bundleContext = FrameworkUtil.getBundle(
+			getClass()
+		).getBundleContext();
+
+		_serviceRegistration = bundleContext.registerService(
+			StagedModelDataHandler.class,
+			new FailingLayoutStagedModelDataHandler(),
+			HashMapDictionaryBuilder.<String, Object>put(
+				"model.class.name", Layout.class.getName()
+			).put(
+				"service.ranking", Integer.MAX_VALUE
+			).build());
+
+		try {
+			_importLayouts(
+				false, larFile, group2.getGroupId(), true, objectDefinition);
```

Comment:
We can `Assert.fail(...)` after this line as well, since this should throw an exception